README.md

```diff
@@ -53,7 +53,9 @@ As with the other tasks, it is also possible to notify failures manually:
 
 Any of the defaults can be over-ridden in `config/deploy.rb`:
 
-    set :slack_channel, '#devops'
+    set :slack_channel, '#devops' 
+    # it's possible usage more channel with comma separated
+    # set :slack_channel, '#devops,#other-channel'
```

Comment:
@Scuottolinx thanks for pull request! Could we be more explicit here and accept an array of strings rather than splitting a single string?  So something like:

```ruby
set :slack_channel, ['#devops', '#other-channel']
```

This would mean we would need to make use of `Array(fetch(:slack_channel)).each` in the cap task itself to ensure that the current `set :slack_channel, '#devops'` continues to work, but it would avoid any unexpected behaviour from splitting a string such as `'#devops, #other-channel'`